### PR TITLE
merge bitcoin#15474...#16774: miscellaneous backports

### PR DESCRIPTION
**Pull request by kittywhiskers**
### Additional Notes

6f98a32403012cb6b02fdc1d8ca85e0fb064057a is based on 1e996cb

### CODE REVIEW

File: Makefile.am [hunk 1]
```diff
@@ -99,7 +99,7 @@ $(OSX_APP)/Contents/PkgInfo:
 
 $(OSX_APP)/Contents/Resources/empty.lproj:
 	$(MKDIR_P) $(@D)
-	@touch $@ 
+	@touch $@
```
Comment (PastaPastaPasta) [inline Makefile.am:102]: unrelated but okay

**Review by PastaPastaPasta — APPROVED** [review]
utACK for merging via merge commit

File: Makefile.am (hunk 1 above)
Comment (kittywhiskers) [reply Makefile.am:102]: IDE does it by default, removes trailing spaces. Neat feature.

Comment (PastaPastaPasta) [reply Makefile.am:102]: Well it shouldn't! :joy:

File: src/qt/trafficgraphwidget.cpp [hunk 2]
```diff
@@ -124,6 +124,7 @@ void TrafficGraphWidget::paintEvent(QPaintEvent *)
 
     const TrafficGraphData::SampleQueue& queue = trafficGraphData.getCurrentRangeQueueWithAverageBandwidth();
 
+    painter.setRenderHint(QPainter::Antialiasing);
```
Comment (UdjinM6) [inline src/qt/trafficgraphwidget.cpp:127]: 16153: this should apply to `drawPath`s only imo, see 1e996cb

**Review by UdjinM6 — CHANGES_REQUESTED** [review]

File: src/qt/trafficgraphwidget.cpp (hunk 2 above)
Comment (kittywhiskers) [reply src/qt/trafficgraphwidget.cpp]: cherry-picked and squashed in latest push, thanks!

**Review by UdjinM6 — APPROVED** [review]
👍 

utACK

**Review by PastaPastaPasta — APPROVED** [review]
reutACK for merging via merge commit

**Pull request merged**